docs/content/reference/resource-schema/core-schema/secretstore/index.md

```diff
@@ -29,7 +29,7 @@ Note that only Kubernetes Secrets are currently supported with more to come in t
 
 | Key | Required | Description | Example |
 |----------|:--------:|-------------|---------|
-| application | y | The ID of the application resource this resource belongs to. | `app.id` |
+| application | n | The ID of the application resource this resource belongs to. Can be an inherited from a global `application` property otherwise required.| `app.id` |
```

Comment:
Was this pulled from the Typespec? What is a "global application property"?

Reply:
cc/ @vishwahiremat

Reply:
No it wasn't pulled from the Typespec but I felt like it might be necessary to let the user know that from my understanding in theory its inheriting the application from a global scope but that also means if it doesn't exist in that scope it does need to be defined.

Reply:
Gloabl scope mean the resource is not part of a environment or an application, so we make the these fields optional. I think we can keep the application description as it was and only change the Required flag to `n`